README.md

```diff
@@ -30,27 +41,44 @@ The following represent a few of the applications that teams developed over the
 
 - [Hero](https://github.com/stanleygjones/hackathon) - A random "killer app" generator inspired by the Wu-Tang Name Generator.
 
+## Example apps from the DFINITY team
+
+- [IC Game Terminal](https://github.com/matthewhammer/ic-game-terminal) - Keyboard input (⌨) and graphical output (📺) for programs on the Internet Computer.
+
+- [Journey](https://github.com/hansl/journey) - An open blog engine written for the Internet Computer.
+
+- [Reversi](https://github.com/ninegua/reversi) - A multiplayer Reversi Game on Internet Computer.
+
+- [Superheroes](https://github.com/enzoh/superheroes) - A simple example that demonstrates how to build a CRUD application on the Internet Computer using Motoko and React.
```

Comment:
```suggestion
- [Superheroes](https://github.com/enzoh/superheroes) - A simple CRUD application that demonstrates using Motoko and React to create your own universe of superheroes that runs on the Internet Computer .
```
This one is a bit awkward. 

I might consider revising all of the descriptions to focus on what they do or what they illustrate.
Manage your own universe of superheroes with a simple application that illustrates how to create, read, update, and delete (CRUD) objects using Motoko andReact.